testsuite/tests/service_mesh/auth/wasm/test_authorities.py

```diff
@@ -4,76 +4,106 @@
 import pytest
 from testsuite.capabilities import Capability
 from testsuite.gateways.wasm import ServiceMeshHttpClient
-
+from testsuite.utils import blame
+from testsuite import rawobj
 
 pytestmark = pytest.mark.required_capabilities(Capability.SERVICE_MESH_WASM)
 
 
 @pytest.fixture(scope="module")
-def client_ingress(api_client, extension) -> ServiceMeshHttpClient:
-    """Returns api client using default ingress_url as ingress to service mesh."""
-    client: ServiceMeshHttpClient = api_client()
-    client.root_url = extension.ingress_url  # redundant, as its default, but included for better readability
-    return client
+# pylint: disable=too-many-arguments
+def custom_extension_and_app(custom_service, custom_app_plan, custom_application,
```

Comment:
The same result should be achievable with only rewriting existing fixtures to be `function` scope

Reply:
Addressed